mobile/app/(app)/tosign/index.tsx

```diff
@@ -1,101 +1,282 @@
+
+              <Ruller />
+
+              <FormItem label="Callback">
+                <TextBodyWhite>{callback}</TextBodyWhite>
+              </FormItem>
+
+              <Ruller />
+
+              <FormItem label="Address">
+                <TextBodyWhite>{bech32Address}</TextBodyWhite>
+              </FormItem>
 
-    return (
-        <>
-            <Layout.Container>
-                <Layout.BodyAlignedBotton>
-                    <Text.Title>{clientName} is requiring permission to {reason}.</Text.Title>
-                    <Spacer space={16} />
-
-                    <Spacer space={16} />
-                    <Text.Body>reason: {reason}</Text.Body>
-                    <Text.Body>callback: {callback}</Text.Body>
-                    <Text.Body>client_name: {clientName}</Text.Body>
-                    <Text.Body>accountName: {accountName}</Text.Body>
-                    <View style={{ flexDirection: 'row', alignItems:'center' }}>
-                        <Text.Body>Address:</Text.Body>
-                        <Text.Caption1>{bech32Address}</Text.Caption1>
-                    </View>
-                    <View style={{ flexDirection: 'row', alignItems:'center' }}>
-                        <Text.Body>Keyinfo:</Text.Body>
-                        <Text.Caption1>{keyInfo?.name}</Text.Caption1>
-                    </View>
-                    <Text.Body>remote: {remote}</Text.Body>
-                    <Text.Body>chainId: {chainId}</Text.Body>
-
-                    <Spacer space={16} />
-
-                    <Button.TouchableOpacity title="Approve" variant="primary" onPress={signTxAndReturnToRequester}></Button.TouchableOpacity>
-                    <Button.TouchableOpacity title="Cancel" variant="primary-red" onPress={onCancel}></Button.TouchableOpacity>
-                </Layout.BodyAlignedBotton>
-            </Layout.Container>
-        </>
-    )
+              <Ruller />
 
+              <FormItem label="Account Name">
+                <TextBodyWhite>{clientName}</TextBodyWhite>
+              </FormItem>
+
+              <Ruller />
+
+              <FormItem label="Keyinfo">
+                <TextBodyWhite>{JSON.stringify(keyInfo?.name)}</TextBodyWhite>
+              </FormItem>
+
+              <Ruller />
+
+              <FormItem label="Remote">
+                <TextBodyWhite>{remote}</TextBodyWhite>
+              </FormItem>
+
+              <Ruller />
+
+              <FormItem label="Chain ID">
+                <TextBodyWhite>{chainId}</TextBodyWhite>
+              </FormItem>
+
+              {/* <FormItem label="Session wanted">
+                <TextBodyWhite>{JSON.stringify(sessionWanted)}</TextBodyWhite>
+              </FormItem>
+
+              <Ruller />
+
+              <FormItem label="Session">
+                <TextBodyWhite>{session ? JSON.stringify(session) : 'undefined'}</TextBodyWhite>
+              </FormItem> */}
+
+              <Ruller />
+
+              <FormItem label="Realms Allowed">
```

Comment:
I suggest to also comment out Realms Allowed, since this is only for account sessions.